appveyor.yml

```diff
@@ -21,6 +21,7 @@ environment:
     - nodejs_version: "17.9"
     - nodejs_version: "18.15"
     - nodejs_version: "19.7"
+    - nodejs_version: "20.11.1"
```

Comment:
```suggestion
    - nodejs_version: "20.11"
```

Reply:
Thanks @aravindvnair99. I reverted that commit